aspnetcore/data/ef-mvc/read-related-data.md

```diff
@@ -164,6 +164,12 @@ Next, if a course was selected, the selected course is retrieved from the list o
 
 [!code-csharp[](intro/samples/cu/Controllers/InstructorsController.cs?range=64-69)]
 
+### Tracking vs no-tracking
+
+No-tracking queries are useful when the results are used in a read-only scenario. They're generally quicker to execute because there's no need to set up the change tracking information. If the entities retrieved from the database don't need to be updated, then a no-tracking query should be used.
+
+In some cases a tracking query is more efficient than a no-tracking query. For more information, see [Tracking vs. No-Tracking Queries](/ef/core/querying/tracking).
+
```

Comment:
```suggestion

```suggestion

In the preceding code, `AsNoTracking` is called because the entities aren't updated in the current context.
```